src/components/app/routes/data/services.js

```diff
@@ -364,6 +365,33 @@ export async function fetchSubscriptions(enterpriseUuid) {
   };
 }
 
+// Notices
+export const fetchNotices = async () => {
+  const authenticatedUser = getAuthenticatedUser();
+  const url = new URL(`${getConfig().LMS_BASE_URL}/notices/api/v1/unacknowledged`);
```

Comment:
nit: any reason not just have this is a static string here, without `new URL`? That way you can just pass `url` to the `.get(...)` function below.